src/lib/php/common-sysconfig.php

```diff
@@ -402,9 +402,9 @@ function Populate_sysconfig()
 
   $variable = "fossologyInstanceUUID";
   $fossologyInstanceUUIDPrompt = _('Fosslogy instance UUID');
-  $fossologyInstanceUUIDDesc = _('Set the fossology instance UUID. default autogenerated value if empty.');
+  $fossologyInstanceUUIDDesc = _('e.g. Instance UUID formate = [a-zA-Z0-9_-]* <br>Set the fossology instance UUID. Used autogenerated value if empty.');
```

Comment:
Teh wording is a bit strange ; why not:
> Set the fossology instance UUID, leave empty to use autogenerated value.

Reply:
Sure, Will update it, Thank you for the suggestion :). Mine was messy even grammatically,